doc/syscalls/00007_analog_comparator.md

```diff
@@ -55,7 +55,7 @@ the index of the last AC module.
 
     **Returns**: `Ok(())` if starting interrupts was succesful.
 
-* ### Command number: `4`
+* ### Command number: `3`
```

Comment:
Just noting for myself: this is not a change per se, but a fix of a documentation bug